03_launch_mgmt_cluster.sh

```diff
@@ -125,11 +125,13 @@ EOF
     fi
     if [ "${IRONIC_BASIC_AUTH}" == "true" ]; then
       sudo mkdir -p /opt/metal3/auth/ironic
+      sudo chown "$USER":"$USER" /opt/metal3/auth/ironic
       cp "${IRONIC_AUTH_DIR}ironic-username" /opt/metal3/auth/ironic/username
       cp "${IRONIC_AUTH_DIR}ironic-password" /opt/metal3/auth/ironic/password
       sudo mkdir -p /opt/metal3/auth/ironic-inspector
-      cp "${IRONIC_AUTH_DIR}ironic-inspector-username" /opt/metal3/auth/ironic-inspector/username
-      cp "${IRONIC_AUTH_DIR}ironic-inspector-password" /opt/metal3/auth/ironic-inspector/password
+      sudo chown "$USER":"$USER" /opt/metal3/auth/ironic-inspector
+      cp "${IRONIC_AUTH_DIR}ironic-username" /opt/metal3/auth/ironic-inspector/username
```

Comment:
@hardys Would you mind explaining why this was changed from inspector files to ironic ones ? (sorry for the delay I was on PTO)